src/click/termui.py

```diff
@@ -604,7 +604,7 @@ def unstyle(text: str) -> str:
 
 def secho(
     message: t.Optional[t.Any] = None,
-    file: t.Optional[t.IO] = None,
+    file: t.Optional[t.IO[t.Any]] = None,
```

Comment:
You could look at using `typing.AnyStr` here, rather than just a blanket `typing.Any`.